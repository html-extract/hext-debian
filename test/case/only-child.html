<div class="only-child div">
  <div class="nope1"></div>
  <div class="nope2"></div>
  <div class="nope3">
    <span class="only-child span"></span>
  </div>
</div>
