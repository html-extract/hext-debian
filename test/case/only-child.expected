{"class":"only-child div"}
{"class":"only-child span"}
